lambdaclass/mirra_backend
# Targeting strategy frontline backline

## Motivation

Game design requires more strategies.
Closes https://github.com/lambdaclass/champions_of_mirra/issues/405

first commit of the pr: https://github.com/lambdaclass/mirra_backend/pull/553/commits/30cdab9c564a4222e950012f92c4154aae5ed2cf

## Summary of changes

- Add new targeting strategies
- Add tests for them

## How to test it?

You can check `battle.md` on how to run battles. To set strategies to "Backline" or "Frontline", you can do so in the `skills.json` file, running `Champions.Config.import_skill_config()`.
Also take a look at the new tests for some examples.
